tasks/config/run.js

```diff
@@ -77,6 +79,22 @@ module.exports = function (grunt) {
       ]
     },
 
+    testUIReleaseServer: {
+      options: {
+        wait: false,
+        ready: /Server running/,
+        quiet: false,
+        failOnError: false
+      },
+      cmd: releaseBinScript,
+      args: [
+        ...stdDevArgs,
+        '--server.port=' + uiConfig.servers.kibana.port,
+        '--elasticsearch.url=' + format(uiConfig.servers.elasticsearch),
+        ...kbnServerFlags,
+      ]
+    },
+
```

Comment:
It's your judgement call I say.  Something like the below isn't all that unwieldy.
```const uiConfig = ...the thing...```
```
 testUIServer: {
      ...uiConfig,
      cmd: binScript
    },

    testUIReleaseServer: {
      ...uiConfig,
      cmd: binScript
    },
```